app/controllers/landing_page_controller.rb

```diff
@@ -175,20 +188,30 @@ def render_landing_page(cid, structure, lp_enabled)
     )
 
     render_to_string :landing_page,
-           locals: { font_path: font_path,
+           locals: { font_path: FONT_PATH,
                      styles: landing_page_styles,
                      javascripts: {
                        location_search: location_search_js
                      },
                      page: denormalizer.to_tree(structure, root: "page"),
-                     sections: denormalizer.to_tree(structure, root: "composition") }
+                     sections: denormalizer.to_tree(structure, root: "composition"),
+                     topbar_props: props,
+                     topbar_props_path: ui_api_topbar_props_path }
   end
 
   def render_not_found(msg = "Not found")
     self.status = 404
     self.response_body = msg
   end
 
+  def topbar_props(community, community_customization, request_path)
+    TopbarHelper.topbar_props(
+      community: community,
+      path_after_locale_change: request_path.gsub(/^\/en/, "").gsub(/^\//, ""),
```

Comment:
Out of scope of this PR, but shouldn't the front-end figure out this itself? In the future, when we add routing to front-end, this won't work anymore.